README.md

```diff
@@ -1,2 +1,55 @@
-# embedding_images_videos
-First microverse project html_css
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Embedding images and videos
+
+> Collaborative project course html and css course Marcelo and Julián
+
+![screenshot](./app_screenshot.png)
+
+First project main curriculum
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
The **simple example steps** are nowhere to be found :smile:

Reply:
We wrote information about how to use our project.